projects/aca-playwright-shared/src/utils/utils.ts

```diff
@@ -46,4 +48,24 @@ export class Utils {
   static formatDate(date: string): string {
     return new Date(date).toLocaleDateString('en-US');
   }
+
+  static async tryLoginUser(loginPage: LoginPage, username: string, password: string, errorMessage?: string) {
+    try {
+      await loginPage.loginUser({ username, password }, { withNavigation: true, waitForLoading: true });
+    } catch (error) {
+      console.error(`${errorMessage}: ${error}`);
+    }
+  }
+
+  static async deleteNodesSitesEmptyTrashcan(nodesApi?: NodesApi, trashcanApi?: TrashcanApi,  errorMessage?: string, sitesApi?: SitesApi, sitesToDelete?: string[], deleteSitesBoolean?: boolean) {
```

Comment:
instead of extra parameter: 'deleteSitesBoolean' I would just use sitesToDelete parameter. 
If somebody passes sitesToDelete then means that person wants to delete some sites. If none sitesToDelete are passed then it means nothing should be deleted. No need for extra flag because there is no sense to pass sitesToDelete in case somebody does not want to delete sites.